crates/librqbit/src/storage/examples/inmemory.rs

```diff
@@ -41,10 +63,15 @@ pub struct InMemoryExampleStorage {
     lengths: Lengths,
     file_infos: FileInfos,
     map: RwLock<HashMap<ValidPieceIndex, InMemoryPiece>>,
+    max_ram_size_per_torrent: usize,
 }
 
 impl InMemoryExampleStorage {
-    fn new(lengths: Lengths, file_infos: FileInfos) -> anyhow::Result<Self> {
```

Comment:
please don't change the example storage. It's there just to show how to implement a custom storage and serves no other purpose.

Deleting pieces after completion is non conformant, and even if done needs to be done properly and probably would span other changes in librqbit which I'm not sure about at least yet.